ext/build_defaults.yaml

```diff
@@ -26,8 +26,8 @@ build_msi:
     repo: 'git://github.com/puppetlabs/hiera.git'
   sys:
     ref:
-      x86: '766c286233138db2140ab3c3cdcc0de9c3041203'
-      x64: '808e292150274f7eaded27445ad33174a1410716'
+      x86: 'refs/tags/1.9.3-p551.4'
+      x64: 'refs/tags/2.0.0.7-x64'
```

Comment:
Right - the naming from the 1.9.3-p551.4 suggests something similar to git describe. For some reason I had the thought that you could actually do that (e.g. commits since tag).